### PR TITLE
Story 0 yarong

### PR DESCRIPTION
**Pull request by yaronyg**
Mostly deleting a bunch of stuff we didn't need and reconnecting the code to use Thali as a normal NPM dependency.
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/thaliproject/postcardapp/pull/16%23issuecomment-128459093%22%2C%20%22https%3A//github.com/thaliproject/postcardapp/pull/16%23issuecomment-128459135%22%2C%20%22https%3A//github.com/thaliproject/postcardapp/pull/16%23discussion_r36328617%22%2C%20%22https%3A//github.com/thaliproject/postcardapp/pull/16%23discussion_r36328729%22%2C%20%22https%3A//github.com/thaliproject/postcardapp/pull/16%23discussion_r36328748%22%2C%20%22https%3A//github.com/thaliproject/postcardapp/pull/16%23discussion_r36328923%22%2C%20%22https%3A//github.com/thaliproject/postcardapp/pull/16%23discussion_r36365380%22%2C%20%22https%3A//github.com/thaliproject/postcardapp/pull/16%23discussion_r36365408%22%2C%20%22https%3A//github.com/thaliproject/postcardapp/pull/16%23discussion_r36365510%22%2C%20%22https%3A//github.com/thaliproject/postcardapp/pull/16%23discussion_r36365534%22%2C%20%22https%3A//github.com/thaliproject/postcardapp/pull/16%23discussion_r36365582%22%2C%20%22https%3A//github.com/thaliproject/postcardapp/pull/16%23discussion_r36444434%22%2C%20%22https%3A//github.com/thaliproject/postcardapp/pull/16%23discussion_r36444564%22%2C%20%22https%3A//github.com/thaliproject/postcardapp/pull/16%23discussion_r36444602%22%2C%20%22https%3A//github.com/thaliproject/postcardapp/pull/16%23discussion_r36444711%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/thaliproject/postcardapp/pull/16%23issuecomment-128459093%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222015-08-06T17%3A57%3A53Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/11578097%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mpodwysocki%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222015-08-06T17%3A58%3A04Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/11578097%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mpodwysocki%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20dbd511ab9051c21e6801492a9ea64e4bf5dd0f4a%20www/jxcore/package.json%204%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/thaliproject/postcardapp/pull/16%23discussion_r36328923%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Multiplex%20not%20needed%20because%20is%20a%20copy%20from%20thali%22%2C%20%22created_at%22%3A%20%222015-08-05T17%3A28%3A32Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/49051%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mattpodwysocki%22%7D%7D%2C%20%7B%22body%22%3A%20%22Fixed%22%2C%20%22created_at%22%3A%20%222015-08-05T23%3A07%3A15Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1480964%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/yaronyg%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20%22%2C%20%22created_at%22%3A%20%222015-08-06T17%3A56%3A44Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/11578097%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mpodwysocki%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20www/jxcore/package.json%3AL10-19%22%7D%2C%20%22Pull%20dbd511ab9051c21e6801492a9ea64e4bf5dd0f4a%20readme.md%2037%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/thaliproject/postcardapp/pull/16%23discussion_r36328748%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Not%20there%20on%20Windows%2C%20perhaps%20PowerShell%20script%20instead%3F%22%2C%20%22created_at%22%3A%20%222015-08-05T17%3A26%3A56Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/49051%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mattpodwysocki%22%7D%7D%2C%20%7B%22body%22%3A%20%22See%20comment%20on%20use%20of%20git%20bash%22%2C%20%22created_at%22%3A%20%222015-08-05T23%3A06%3A37Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1480964%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/yaronyg%22%7D%7D%2C%20%7B%22body%22%3A%20%22Perfect%21%20%3A%2B1%3A%20%20%20Would%20love%20to%20see%20people%20use%20cmder%20by%20default%20on%20Windows%20quite%20honestly%21%22%2C%20%22created_at%22%3A%20%222015-08-06T17%3A55%3A21Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/11578097%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mpodwysocki%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20readme.md%3AL13-33%22%7D%2C%20%22Pull%20dbd511ab9051c21e6801492a9ea64e4bf5dd0f4a%20readme.md%2028%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/thaliproject/postcardapp/pull/16%23discussion_r36328617%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Mac%20and%20Linux%22%2C%20%22created_at%22%3A%20%222015-08-05T17%3A25%3A50Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/49051%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mattpodwysocki%22%7D%7D%2C%20%7B%22body%22%3A%20%22Fixed.%22%2C%20%22created_at%22%3A%20%222015-08-05T23%3A04%3A45Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1480964%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/yaronyg%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20%22%2C%20%22created_at%22%3A%20%222015-08-06T17%3A57%3A40Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/11578097%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mpodwysocki%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20readme.md%3AL13-33%22%7D%2C%20%22Pull%20dbd511ab9051c21e6801492a9ea64e4bf5dd0f4a%20readme.md%2031%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/thaliproject/postcardapp/pull/16%23discussion_r36328729%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Not%20there%20on%20Windows%2C%20perhaps%20PowerShell%20script%20instead%3F%22%2C%20%22created_at%22%3A%20%222015-08-05T17%3A26%3A49Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/49051%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mattpodwysocki%22%7D%7D%2C%20%7B%22body%22%3A%20%22Actually%20I%20added%20the%20line%20at%20the%20bottom%3A%5Cr%5Cn%5Cr%5CnOn%20Windows%20one%20needs%20to%20use%20%5BGit%20Bash%5D%28https%3A//git-scm.com/download/win%29%20or%20equivalent%20to%20run%20the%20above%20commands.%22%2C%20%22created_at%22%3A%20%222015-08-05T23%3A05%3A11Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1480964%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/yaronyg%22%7D%7D%2C%20%7B%22body%22%3A%20%22And%20yes%2C%20I%20did%20check%20that%20all%20the%20various%20commands%20do%20work.%20E.g.%20curl%2C%20unzip%20and%20find%20are%20all%20built%20in.%22%2C%20%22created_at%22%3A%20%222015-08-05T23%3A06%3A21Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1480964%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/yaronyg%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20sounds%20good%20enough%21%22%2C%20%22created_at%22%3A%20%222015-08-06T17%3A56%3A27Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/11578097%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mpodwysocki%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20readme.md%3AL13-33%22%7D%7D%2C%20%22approved%22%3A%20%7B%22https%3A//github.com/mpodwysocki%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/11578097%3Fv%3D3%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/mpodwysocki'></a>
- [x] <a href='#crh-comment-Pull dbd511ab9051c21e6801492a9ea64e4bf5dd0f4a readme.md 28'></a> &nbsp;<b><a href='https://github.com/thaliproject/postcardapp/pull/16#discussion_r36328617'>File: readme.md:L13-33</a></b>
- <a href='https://github.com/mattpodwysocki'></a> Mac and Linux
- <a href='https://github.com/yaronyg'></a> Fixed.
- [x] <a href='#crh-comment-Pull dbd511ab9051c21e6801492a9ea64e4bf5dd0f4a readme.md 31'></a> &nbsp;<b><a href='https://github.com/thaliproject/postcardapp/pull/16#discussion_r36328729'>File: readme.md:L13-33</a></b>
- <a href='https://github.com/mattpodwysocki'></a> Not there on Windows, perhaps PowerShell script instead?
- <a href='https://github.com/yaronyg'></a> Actually I added the line at the bottom:
  On Windows one needs to use [Git Bash](https://git-scm.com/download/win) or equivalent to run the above commands.
- <a href='https://github.com/yaronyg'></a> And yes, I did check that all the various commands do work. E.g. curl, unzip and find are all built in.
- <a href='https://github.com/mpodwysocki'></a> :+1: sounds good enough!
- [x] <a href='#crh-comment-Pull dbd511ab9051c21e6801492a9ea64e4bf5dd0f4a readme.md 37'></a> &nbsp;<b><a href='https://github.com/thaliproject/postcardapp/pull/16#discussion_r36328748'>File: readme.md:L13-33</a></b>
- <a href='https://github.com/mattpodwysocki'></a> Not there on Windows, perhaps PowerShell script instead?
- <a href='https://github.com/yaronyg'></a> See comment on use of git bash
- <a href='https://github.com/mpodwysocki'></a> Perfect! :+1:   Would love to see people use cmder by default on Windows quite honestly!
- [x] <a href='#crh-comment-Pull dbd511ab9051c21e6801492a9ea64e4bf5dd0f4a www/jxcore/package.json 4'></a> &nbsp;<b><a href='https://github.com/thaliproject/postcardapp/pull/16#discussion_r36328923'>File: www/jxcore/package.json:L10-19</a></b>
- <a href='https://github.com/mattpodwysocki'></a> Multiplex not needed because is a copy from thali
- <a href='https://github.com/yaronyg'></a> Fixed

<a href='https://www.codereviewhub.com/thaliproject/postcardapp/pull/16?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/thaliproject/postcardapp/pull/16?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/thaliproject/postcardapp/pull/16?approve=0'></a>&nbsp;<a href='https://github.com/thaliproject/postcardapp/pull/16'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/thaliproject/postcardapp/pull/16" target="_blank">ReviewNinja

**Comment by yaronyg** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/thaliproject/postcardapp/pull/16'>Refresh</a>.<a href='#crh-update-none'></a>

File: readme.md [hunk 1]
```diff
@@ -13,26 +13,20 @@ plugin it depends on.
...
-
-
-
-
-
-
+1. Install JXcore from [here](http://jxcore.com/downloads) 
+2. Install Cordova 
+ 1. Windows - `jx install cordova -g`
+ 2. Linux - `sudo jx install cordova -g`
```
Comment (mattpodwysocki) [inline readme.md]: Mac and Linux

File: readme.md [hunk 2]
```diff
@@ -13,26 +13,20 @@ plugin it depends on.
...
-
-
-
+1. Install JXcore from [here](http://jxcore.com/downloads) 
+2. Install Cordova 
+ 1. Windows - `jx install cordova -g`
+ 2. Linux - `sudo jx install cordova -g` 
+ 
+```
+curl https://codeload.github.com/thaliproject/postcardapp/zip/story_0 > thali.zip
```
Comment (mattpodwysocki) [inline readme.md:22]: Not there on Windows, perhaps PowerShell script instead?

File: readme.md [hunk 3]
```diff
@@ -13,26 +13,20 @@ plugin it depends on.
...
+unzip thali.zip
+cd postcardapp-story_0
+cordova platform add android
+cd www/jxcore
+jx npm install --production
+find . -name "*.gz" -delete
```
Comment (mattpodwysocki) [inline readme.md:28]: Not there on Windows, perhaps PowerShell script instead?

File: www/jxcore/package.json [hunk 4]
```diff
@@ -10,8 +10,9 @@
     "express-pouchdb": "^0.14.2",
     "leveldown": "^1.3.0",
     "leveldown-mobile": "^1.0.7",
+    "multiplex": "^6.2.2",
```
Comment (mattpodwysocki) [inline www/jxcore/package.json]: Multiplex not needed because is a copy from thali

File: readme.md (hunk 1 above)
Comment (yaronyg) [inline readme.md]: Fixed.

File: readme.md (hunk 2 above)
Comment (yaronyg) [inline readme.md:22]: Actually I added the line at the bottom:

On Windows one needs to use [Git Bash](https://git-scm.com/download/win) or equivalent to run the above commands.

Comment (yaronyg) [inline readme.md:22]: And yes, I did check that all the various commands do work. E.g. curl, unzip and find are all built in.

File: readme.md (hunk 3 above)
Comment (yaronyg) [inline readme.md:28]: See comment on use of git bash

File: www/jxcore/package.json (hunk 4 above)
Comment (yaronyg) [inline www/jxcore/package.json]: Fixed

File: readme.md (hunk 3 above)
Comment (mpodwysocki) [inline readme.md:28]: Perfect! :+1:   Would love to see people use cmder by default on Windows quite honestly!

File: readme.md (hunk 2 above)
Comment (mpodwysocki) [inline readme.md:22]: :+1: sounds good enough!

File: www/jxcore/package.json (hunk 4 above)
Comment (mpodwysocki) [inline www/jxcore/package.json]: :+1:

File: readme.md (hunk 1 above)
Comment (mpodwysocki) [inline readme.md]: :+1:

**Comment by mpodwysocki** [comment]
:+1:

**Comment by mpodwysocki** [comment]
:shipit:

**Pull request merged**